utils/mdldec/qc.c

```diff
@@ -371,13 +371,13 @@ static void WriteBodyGroupInfo( FILE *fp )
 
 			if( !Q_strncmp( model->name, "blank", 5 ) )
```

Comment:
Is it should catch everything that starts with `blank`?

Reply:
Yep.